src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Hydrography/ChangeEncDisplaySettings/ChangeEncDisplaySettings.xaml.cs

```diff
@@ -110,10 +111,17 @@ private void UpdateDisplaySettings()
             else { globalMarinerSettings.PointSymbolizationType = EncPointSymbolizationType.Simplified; }
         }
 
-        private void Button_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        private void Setting_Checked(object sender, Windows.UI.Xaml.RoutedEventArgs e)
         {
             // Apply display settings
-            UpdateDisplaySettings();
+            try
+            {
+                UpdateDisplaySettings();
+            }
+            catch (NullReferenceException)
+            {
+                // The sample wasn't ready
```

Comment:
How barbaric! I'm guessing that UWP attaches our checked event handler before setting IsChecked in XAML? I was hoping that there'd be a cleaner way to address this...